src/components/DevelopmentGuide/ConditionalViewport.mdx

```diff
@@ -1,10 +1,15 @@
 import { Meta, Source } from "@storybook/blocks";
 
+import ComponentDocsHeader from "../../utils/components/ComponentDocsHeader";
 import Link from "../Link/Link";
 
 <Meta title="Development Guide/Conditional Viewport" />
 
-# Conditional Viewport
+<ComponentDocsHeader
+  category="Development Guide"
+  componentName="Conditional Viewport"
+  summary="Recommendations for mitigating the iOS auto-zoom"
```

Comment:
Nit: elsewhere we say "preventing" which is a little different than "mitigating"

Reply:
I was unsure about this one. I like `preventing`. Thanks. Will change.